WebInkEnhancement/explainer.md

```diff
@@ -150,5 +155,9 @@ Due to uncertainty around the correct execution when `setLastRenderedPoint` is c
 
 Instead of providing `setLastRenderedPoint` with a PointerEvent, just providing x and y values is also an option. It was decided that a trusted pointer event would likely be the better option though, as then we can have easier access to the pointer ID and the web developer doesn't have to put extra thought into the position of the ink.
 
+Providing the presenter with the canvas allows the boundaries of the drawing area to be determined. This is necessary so that points and ink outside of the desired area aren't drawn when points are being forwarded. If no canvas is provided, then the root layer size will be used.

```

Comment:
I think we may want this to be the containing viewport instead of root layer size. Namely due to the fact that OOPIF's won't have access to the root layer.

Reply:
Makes sense, fixed that.